notebooker/web/static/notebooker/index.js

```diff
@@ -32,59 +32,75 @@ function updateNavigationPanel(currentFolder){
 }
 
 
+function entryAfterLevel(path, level){
+    while(level > 0) {
+        path = path.substring(path.indexOf('/') + 1)
```

Comment:
Will this be Windows-safe? (I think yes, but worth checking?)

Reply:
hmmm I think so based on some code inspection but it will be a bit of a pain to try it out on Windows, have you done it before?